internal/cmd/freeze-api/convert.go

```diff
@@ -58,7 +56,7 @@ func (r *{{ $t.TypeName }}) ConvertTo(to interface{}) {
 }
 
 var convert_{{ $t.TypeName }}_To_{{ $toPackage }}_{{ $t.TypeName}} = func(from *{{ $t.TypeName }}, to *{{ $toPackage}}.{{ $t.TypeName }}) {
-	{{ if $t.Simple }} *to = *(*{{ $toPackage }}.{{ $t.TypeName }})(unsafe.Pointer(from)){{ else }}panic("complex conversion not supported yet"){{ end }}
+	{{ if $t.Simple }}*to = *(*{{ $toPackage }}.{{ $t.TypeName }})(unsafe.Pointer(from)){{ else }}panic("complex conversion not supported yet"){{ end }}
 }
 
 // ConvertFrom converts the receiver to a *{{ $toPackage }}.{{ $t.TypeName }}.
```

Comment:
If my other, related comments make sense, then this should be "converts the receiver to a *{{ $t.TypeName }}."